src/routes/PrimaryLayout/PrimaryLayout.js

```diff
@@ -344,6 +347,8 @@ export default class PrimaryLayout extends Component {
             <Route path={`/:context(all|public)/:view(events)/${OPTIONAL_POST_MATCH}`} component={Events} />
             <Route path={`/:context(all|public)/:view(map)/${OPTIONAL_POST_MATCH}`} component={MapExplorer} />
             <Route path={`/:context(all|public)/:view(map)/${OPTIONAL_GROUP_MATCH}`} component={MapExplorer} />
+            {/* Need to actually point to the correct page component in the route below */}
```

Comment:
What does this comment mean? Is it still relevant?

Reply:
No, thanks for the catch!